pkg/test/istio.io/tasks/security/mtls-migration/mtls-migration_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+package tests
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/istio.io/examples"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+
+	"istio.io/istio/pkg/test/framework/components/istio"
+)
+
+var (
+	ist istio.Instance
+)
+
+func TestMain(m *testing.M) {
+	framework.NewSuite("mtls-migration", m).
+		SetupOnEnv(environment.Kube, istio.Setup(&ist, nil)).
+		RequireEnvironment(environment.Kube).
+		Run()
+}
+
+//https://istio.io/docs/tasks/security/mtls-migration/
```

Comment:
Do we have some way to explicitly correlate these to the docs other than just a comment? Or plans to do so in the future

Because right no we have scripts like `kubectl get destinationrule --all-namespaces` which aren't very useful to test unless that is the same command that is actually in the docs. I know there was some talk about either having the tests pull scripts directly from the docs or the docs pull directly from the test